src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -363,22 +362,46 @@ else if (abbrStarted) {
     /**
      * Get Abbreviation if it is illegal.
      * @param str name
-     * @param beginIndex begin index
-     * @param endIndex end index
-     * @return true is abbreviation is bigger that required and not in ignore list
+     * @param beginIndex begin index, inclusive
+     * @param endIndex end index, inclusive
```

Comment:
same here about `inclusive` for `endIndex`.
It isn't inclusive since you do `endIndex - beginIndex` instead of `endIndex - beginIndex + 1`.